pkg/scripts/testdata/TestKubeadmFlatcar-force.golden

```diff
@@ -62,13 +47,43 @@ EOF
 sudo systemctl force-reload systemd-journald
 
 
-sudo systemctl restart docker
-
 sudo mkdir -p /opt/cni/bin /etc/kubernetes/pki /etc/kubernetes/manifests
 curl -L "https://github.com/containernetworking/plugins/releases/download/v0.8.7/cni-plugins-linux-${HOST_ARCH}-v0.8.7.tgz" |
 	sudo tar -C /opt/cni/bin -xz
 
 RELEASE="v1.17.4"
+MAJOR_RELEASE="$(cut -d '.' -f 1,2 <<< ${RELEASE}).0"
+
+curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/${MAJOR_RELEASE}/crictl-${MAJOR_RELEASE}-linux-${HOST_ARCH}.tar.gz |
+	sudo tar -C /opt/bin -xz
```

Comment:
The crictl (cri-tools) are not part of the Kubernetes release lifecycle (i.e. they are released separately). This approach will not work for newly released Kubernetes versions. For example, once Kubernetes 1.22 goes out, it might take some time until the cri-tools team doesn't release cri-tools v1.22. If someone would try to provision Kubernetes 1.22 before cri-tools v1.22 is out, the cluster provisioning would fail.

As you can use any cri-tools version with any Kubernetes version, I would prefer if we would bind this version, instead of depending on the Kubernetes version.